src/main/webapp/scripts/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Check if user is logged in
+ */
+function authUser() {
+    fetch('/login')
+      .then(response => response.json())
+      .then(userAuthInfo => {
+        let loginButton = document.getElementById('google-login-button');
+        let loginButtonText = document.getElementById('google-login-button-text');
```

Comment:
nit: these should be `const` since the value shouldn't be reassigned

Reply:
Will fix, thanks